### PR TITLE
Update image.md

### PR DESCRIPTION
**Pull request by thomasajai001**
Corrected grammatical mistake in tip

### CODE REVIEW

File: docs/image.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ blurRadius: the blur radius of the blur filter added to the image.
 | ------ |
 | number |
 
-> Tip : IOS you will need to increase `blurRadius` more than `5`
+> Tip : On IOS, you will need to increase `blurRadius` by more than `5`
```
Comment (Simek) [inline docs/image.md:248]:
```suggestion
> Tip: On IOS, you will need to increase `blurRadius` by more than `5`.
```

**Review by Simek — APPROVED** [review]
Hi @thomasajai001, thank you for the correction! 👍 

Please apply the suggested formatting changes, besides that, this LGTM.

File: docs/image.md (hunk 1 above)
Comment (thomasajai001) [reply docs/image.md]: Okay, thanks for the suggestion.

Comment (Simek) [reply docs/image.md]: Thank you for the quick update, let's ship it! 🙂

**Pull request merged**